server/trips/tripmodel.js

```diff
@@ -1,24 +1,20 @@
 var mongoose = require('mongoose');
-​
-var Schema = mongoose.Schema;
-​
-var tripSchema = new Schema({
+
+var tripSchema = new mongoose.Schema({
   tripName: String,
   members: {
     type: Array,
-    default:[]
+    default:[],
```

Comment:
This is the last property in the object, so the comma should not have been added.